scp/cc/cpio/client_providers/role_credentials_provider/src/BUILD.bazel

```diff
@@ -23,10 +24,13 @@ cc_library(
             "//:aws_platform": [
                 "//scp/cc/cpio/client_providers/role_credentials_provider/src/aws:aws_role_credentials_provider_lib",
             ],
+            "//:azure_platform": [
+                "//scp/cc/cpio/client_providers/role_credentials_provider/src/gcp:gcp_role_credentials_provider_lib",
+            ],
```

Comment:
This is because we haven't implemented our version. We will Azure our version in a future PR.